### DISCUSSION TITLE
Restoration failed after 1 iteration

### DISCUSSION THREAD

**Discussion by TheSimplyRed** [coin-or/Ipopt#651] [Q&A]
Hi,
For some default values when I use "warm_start_init_point", I have a restoration failed message. 
in the log I have:

> delta_x perturbation is becoming too large: 1.000000e+022
PerturbForSingularity can't be done
The linear system could not be solved for the affine step!
The mu oracle could not compute a new value of the barrier parameter.
Barrier parameter could not be updated!
Fallback option activated in BacktrackingLineSearch!
WARNING: Problem in step computation; switching to emergency mode.

What can I do to avoid that delta_x perturbation becomes too large?

Thanks

**Comment by svigerske** [comment]
From this info, I cannot say why delta_x becomes 1e22. You can change the meaning of "large" by setting option [max_hessian_perturbation](https://coin-or.github.io/Ipopt/OPTIONS.html#OPT_max_hessian_perturbation), though.
